configs/mainnet.json

```diff
@@ -477,5 +477,23 @@
   },
   "spot_pool_config": {
     "show_apr_tooltip": true
-  }
+  },
+  "transfer_quick_select_tokens": [
+    {
+      "show": "brdg/4b90a8d1225d11ed7af0661ff7309714189a527302892cad03ace7615d417d38",
+      "real" : "cgt/1"
+    },
+    {
+      "show": "swth",
+      "real" : "swth"
+    },
+    {
+      "show": "eth.1.2.942d87",
+      "real" : "eth.1.2.942d87"
+    },
+    {
+      "show": "brdg/a02afc2c1edf77cc023eefa25fc036c184612faf9365cda9c1daa3b1675ebf8f",
+      "real" : "brdg/a02afc2c1edf77cc023eefa25fc036c184612faf9365cda9c1daa3b1675ebf8f"
```

Comment:
![image](https://github.com/user-attachments/assets/d76c174e-8a89-4213-bbba-8a860e906465)

think its just a matter of label and the actual token selected. modify the field to show the label, add `icon_denom` to fetch the correct icon and add `quick_select_denom` to link to the intended token after selecting

would be good to add priority to the fields also so the UI can sort by priority

```
{
  "priority": "1",
  "label": "USDC",
  "icon_denom":  "{any usdc denom that returns the usdc icon}",
  "quick_select_denom": "cgt/1",
}, {
...
}